packages/0xcert-ethereum-xcert-contracts/src/tests/xcert.test.ts

```diff
@@ -122,24 +123,24 @@ spec.test('throws when trying to create an xcert to zero address', async (ctx) =
   await ctx.reverts(() => xcert.instance.methods.create(zeroAddress, id, imprint).send({ from: owner }), '006001');
 });
 
-spec.test('corectly assigns create ability', async (ctx) => {
+spec.test('corectly grants create ability', async (ctx) => {
   const xcert = ctx.get('xcert');
   const owner = ctx.get('owner');
   const bob = ctx.get('bob');
  
-  const logs =  await xcert.instance.methods.assignAbilities(bob, 2).send({ from: owner });
-  ctx.not(logs.events.AssignAbilities, undefined);
+  const logs =  await xcert.instance.methods.grantAbilities(bob, XcertAbilities.CREATE_ASSET).send({ from: owner });
+  ctx.not(logs.events.GrantAbilities, undefined);
 
   const bobHasAbility1 = await xcert.instance.methods.isAble(bob, 2).call();
   ctx.is(bobHasAbility1, true);
 });
 
-spec.test('throws when a third party tries to assign create ability', async (ctx) => {
+spec.test('throws when a third party tries to grants create ability', async (ctx) => {
```

Comment:
`spec.test('throws when a third party tries to grant create ability', async (ctx) => {`